elements/rh-progress-stepper/demo/compact-vertical.html

```diff
@@ -24,6 +17,13 @@
     description="This step has failed and needs to be retried">
   </rh-progress-step>
 
+  <rh-progress-step
+    state="active"
+    label="Active Step"
```

Comment:
if this renders visible test, that's fine. if it's only for SR, we've used the conventional attribute `accessible-label`

Reply:
The label attribute does render visible text, but can also be overridden by a `label` slotted element.